docs/FAQ.md

```diff
@@ -46,11 +188,27 @@ engine components you desire, provide any necessary configurations, and build
 an instance of the loader. After that, you configure your system boot to launch
```

Comment:
s/After that, you/Next,/

Edited-by: Amanda Ferguson [af.jenks@gmail.com](mailto:af.jenks@gmail.com)